lib/foreman_remote_execution_core/script_runner.rb

```diff
@@ -120,7 +121,12 @@ def with_disconnect_handling
     end
 
     def close
+      run_sync("rm -rf #{remote_command_dir}") unless @session.nil? || @session.closed?
```

Comment:
We should put the path into quotes, so the users won't wipe their systems if they set remote_working_dir to something with a space